src/ConsoleMode.ps1

```diff
@@ -85,19 +90,29 @@ function Set-ConsoleMode
         $StandardInput
     )
 
-    if ($ANSI)
-    {
-        $outputMode = [NativeConsoleMethods]::GetConsoleMode($false)
-        $null = [NativeConsoleMethods]::SetConsoleMode($false, $outputMode -bor [ConsoleModeOutputFlags]::ENABLE_VIRTUAL_TERMINAL_PROCESSING)
+    begin {
```

Comment:
@rkeithhill What's the advantage of splitting this into `begin`/`end`?

Reply:
In this particular case, there is no advantage.  Then again, there is no harm either.  But I can simplify this if you'd like.

Reply:
No change necessary unless you find the alternative easier to read. Just curious if I was missing something.